### PR TITLE
[ID-38] Core UI - Bottom popup alert dialog

### PR DESCRIPTION
**Pull request by hophongit98**
### https://nartus.atlassian.net/browse/ID-38

### CODE REVIEW

File: packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                    ?.copyWith(color: NartusColor.dark),
+              )),
+          Padding(
+            padding: const EdgeInsets.only(bottom: NartusDimens.padding30),
+            child: NartusButton.primary(
+              label: primaryButtonText,
+              onPressed: onPrimaryButtonSelected,
+            ),
+          ),
+          if (secondaryButtonText != null)
```
Comment (suesitran) [inline packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart:75]: It should have another option for Text button as well.

File: packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+            borderRadius: BorderRadius.vertical(top: Radius.circular(20))),
+        builder: (BuildContext builder) {
+          return NartusBottomSheet(
+            iconPath: iconPath,
+            title: title,
+            content: content,
+            primaryButtonText: primaryBtnText,
+            onPrimaryButtonSelected: onPrimaryBtnSelected,
+            secondaryButtonText: secondaryBtnText,
+            onSecondButtonSelected: onSecondaryBtnSelected,
```
Comment (suesitran) [inline packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart]: Need another option for Text button as well

File: packages/nartus_ui_package/test/widgets/bottom_sheet/nartus_bottom_sheet.dart [hunk 3]
```diff
@@ -0,0 +1,117 @@
+import 'package:flutter_svg/flutter_svg.dart';
```
Comment (suesitran) [inline packages/nartus_ui_package/test/widgets/bottom_sheet/nartus_bottom_sheet.dart]: This file should have suffix as _test

File: packages/nartus_ui_package/test/widgets/bottom_sheet/nartus_bottom_sheet.dart [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            count++;
+          });
+
+      await widgetTester.wrapMaterialAndPump(btmSheetWithIcon);
+
+      expect(find.text('title'), findsOneWidget);
+      expect(find.text('content'), findsOneWidget);
+      expect(find.text('primaryButtonText'), findsOneWidget);
+      expect(find.text('secondaryButtonText'), findsNothing);
+      expect(count, 1);
```
Comment (suesitran) [inline packages/nartus_ui_package/test/widgets/bottom_sheet/nartus_bottom_sheet.dart]: There's no tap action on the button yet, so the count he can't be 1 yet

File: packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+        isDismissible: isDismissible,
+        shape: const RoundedRectangleBorder(
+            borderRadius: BorderRadius.vertical(top: Radius.circular(20))),
+        builder: (BuildContext builder) {
+          return NartusBottomSheet(
+            iconPath: iconPath,
+            title: title,
+            content: content,
+            primaryButtonText: primaryBtnText,
+            onPrimaryButtonSelected: onPrimaryBtnSelected,
```
Comment (suesitran) [inline packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart]: Please use the name onPrimaryButtonPressed, onSecondaryButtonPress, and onTextButtonPressed

**Review by suesitran — CHANGES_REQUESTED** [review]

File: packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart (hunk 1 above)
Comment (hophongit98) [reply packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart:75]: Based on the ticket (content and discussion), The bottom sheet has only two buttons. Should we update the ticket?

Comment (hophongit98) [reply packages/nartus_ui_package/lib/widgets/bottom_sheet/nartus_bottom_sheet.dart:75]: updated

**Review by suesitran — APPROVED** [review]
LGTM

**Pull request merged**